bin/stacks/routing-caching-stack.ts

```diff
@@ -70,9 +91,41 @@ export class RoutingCachingStack extends cdk.NestedStack {
 
     this.poolCacheBucket.grantReadWrite(poolCachingLambda);
 
+    const ipfsPoolCachingLambda = new aws_lambda_nodejs.NodejsFunction(
+      this,
+      'IpfsPoolCacheLambda',
+      {
+        role: lambdaRole,
+        runtime: aws_lambda.Runtime.NODEJS_14_X,
+        entry: path.join(__dirname, '../../lib/cron/cache-pools-ipfs.ts'),
+        handler: 'handler',
+        timeout: Duration.seconds(600),
+        memorySize: 1024,
+        bundling: {
+          minify: true,
+          sourceMap: true,
+        },
+        layers: [
+          aws_lambda.LayerVersion.fromLayerVersionArn(
+            this,
+            'InsightsLayerPoolsIPFS',
+            `arn:aws:lambda:${region}:580247275435:layer:LambdaInsightsExtension:14`
+          ),
+        ],
+        tracing: aws_lambda.Tracing.ACTIVE,
+        environment: {
+          PINATA_API_KEY: pinata_key,
+          PINATA_API_SECRET:  pinata_secret,
+          ROLE_ARN: route53Arn!,
+          HOSTED_ZONE: hosted_zone,
+          STAGE: stage
+        },
+      }
+    );
+
     new aws_events.Rule(this, 'SchedulePoolCache', {
       schedule: aws_events.Schedule.rate(Duration.minutes(2)),
-      targets: [new aws_events_targets.LambdaFunction(poolCachingLambda)],
+      targets: [new aws_events_targets.LambdaFunction(poolCachingLambda), new aws_events_targets.LambdaFunction(ipfsPoolCachingLambda)],
     });
 
     const lambdaAlarmErrorRate = new aws_cloudwatch.Alarm(this, 'RoutingAPI-PoolCacheToS3LambdaError', {
```

Comment:
Would be great to add an alarm for the new lambda (also only created if on beta)